strategies/candlesticks/white_shark.js

```diff
@@ -61,6 +60,7 @@ class WhiteShark {
   }
 
   ticker(value) {
+    //this.logger.debug(` - Value:${value}`);
```

Comment:
Codacy has a fix for the issue: [Expected exception block, space or tab after '//' in comment.](https://app.codacy.com/gh/amcalabretta/botbase/pullRequest?prid=7628479)
```suggestion
    // this.logger.debug(` - Value:${value}`);
```